reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md

```diff
@@ -96,106 +96,6 @@ Beginning in Windows PowerShell 3.0, the Enable-PSRemoting cmdlet can enable
 remoting on client and server versions of Windows on private, domain, and
 public networks.
 
-On server versions of Windows with private and domain networks, the
```

Comment:
It appears that part of the file got pasted in here

Reply:
Weird. Thanks for the fix.